meson.md

```diff
@@ -0,0 +1,54 @@
+# Building with Meson
+
+### Developer build
+
+**Install build tools:** `pip install meson-python cython pythran`
+
+**Generate ninja make files:** `meson build --prefix=$PWD/build`
+
+**Compile:** `ninja -C build`
+
+**Install:** `meson install -C build`
+
+The install step copies the necessary Python files into the build dir to form a complete package.
+Do not skip this step, or the package won't work.
+
+To use the package, add it to your PYTHONPATH:
+
+```
+export PYTHONPATH=${PWD}/build/lib64/python3.10/site-packages
```

Comment:
On my system this path is different: `build/lib/python3.10/site-packages`...?

Reply:
Yes, the `dev.py` tool takes this into consideration, but these notes obviously cannot cover all platform intricacies.